tools/test/test262/test262.py

```diff
@@ -319,9 +320,16 @@ def GetDriverSource(self):
 
   def GetSource(self):
     if self.IsRaw():
-        return self.test
-
-    source = self.test
+        return self.source
+
+    source = self.source
+    # if test is hashbang test
+    if len(source) > 1 and (source[0] == '#' or source[0] == '\\'):
+      lines = source.splitlines()
+      if self.strict_mode:
+        lines.insert(1, 'var strict_mode = true;')
+        lines.insert(1, '"use strict";')
```

Comment:
Why did you add these two lines for strict mode?

Reply:
every test262 test cases doesn't have "use strict" directive at begining of its soruce code
so we have to append "use strict" directive if we want to test on strict-mode
you can see same thing on 335 line
```
    if self.strict_mode:
      source = '"use strict";\nvar strict_mode = true;\n' + source
```

Reply:
I see :)